src/snowflake/snowpark/_internal/analyzer/analyzer.py

```diff
@@ -598,9 +584,6 @@ def do_resolve_with_resolved_children(
             )
 
         if isinstance(logical_plan, Pivot):
-            if len(logical_plan.aggregates) != 1:
-                raise ValueError("Only one aggregate is supported with pivot")
-
```

Comment:
Why not add a negative test instead of removing this check?

Reply:
forgot this one - already checked here https://github.com/snowflakedb/snowpark-python/blob/e3117c91a91c678d41ea77824bdd55272429b388/src/snowflake/snowpark/relational_grouped_dataframe.py#L177